packages/manager/modules/pci/src/projects/project/private-registry/delete/delete.html

```diff
@@ -8,8 +8,8 @@
         data-heading="{{:: 'private_registry_delete' | translate }}"
         data-primary-label="{{:: 'private_registry_common_delete' | translate }}"
         data-secondary-label="{{:: 'common_cancel' | translate }}"
-        data-secondary-action="$ctrl.goBack()"
-        data-on-dismiss="$ctrl.goBack()"
+        data-secondary-action="$ctrl.goBack(); $ctrl.trackClick('PCI_PROJECTS_PRIVATEREGISTRY_DELETE_CANCEL')"
+        data-on-dismiss="$ctrl.goBack(); $ctrl.trackClick('PCI_PROJECTS_PRIVATEREGISTRY_DELETE_CLOSE')"
```

Comment:
Maybe these two calls should be set into one public method, it should help readability